src/appleseed/renderer/modeling/aov/normalaov.cpp

```diff
@@ -125,7 +147,7 @@ namespace
 
         virtual const char** get_channel_names() const override
         {
-            static const char* ChannelNames[] = {"X", "Y", "Z"};
+            static const char* ChannelNames[] = {"R", "G", "B"};
```

Comment:
What's the rationale behind this change?

Reply:
It makes it much easier to use the resulting images in compositing applications.